modules/mod_core.c

```diff
@@ -6136,10 +6136,11 @@ MODRET core_feat(cmd_rec *cmd) {
   /* Sort the features, for a prettier output. */
   qsort(feats->elts, feats->nelts, sizeof(char *), feat_cmp);
 
-  pr_response_add(R_211, "%s", _("Features:"));
+  feat = pstrcat(cmd->tmp_pool, "Features:\n ", NULL);
   for (i = 0; i < feats->nelts; i++) {
-    pr_response_add(R_DUP, "%s", ((const char **) feats->elts)[i]);
+    feat = pstrcat(cmd->tmp_pool, feat, ((const char **) feats->elts)[i], "\n ", NULL);
```

Comment:
`FEAT` lines should be terminated using `\r\n`, not just `\n`'; the adding of CRLF characters should be done properly by `pr_response_add()`.  So this patch appears to be _removing_ the use of CRLF, in favor of just LF...?